packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -96,7 +99,18 @@ const useStyles = makeStyles(({ palette, spacing }) => ({
   },
 
   hiddenDetailsTileWrapper: {
-    marginBottom: spacing(3)
+    ...flexRowCentered(),
+    marginBottom: spacing(4)
+  },
+
+  hiddenTrackLabel: {
+    marginTop: spacing(1),
+    marginLeft: spacing(2),
+    color: palette.accentOrange,
+    fontFamily: typography.fontByWeight.demiBold,
+    fontSize: 14,
+    letterSpacing: 2,
+    textTransform: 'uppercase'
```

Comment:
It seems weird that we have variants for the Text component, but still need to add custom styles in a lot of places. Maybe we should revisit the props to make it more flexible?

Reply:
I agree, we actually deprecated our Text component for this reason. Decided to use styles exclusively bc it will be the most flexible, and is still convenient with the `typography` stuff Dylan added

So going forward let's just use `Text` from `react-native`